src/main/java/org/generationcp/middleware/operation/builder/StandardVariableBuilder.java

```diff
@@ -79,6 +90,21 @@ public StandardVariable create(final int standardVariableId, final String progra
 		return standardVariables;
 	}
 
+	public List<StandardVariable> createStandardVariableWithAliasAsName(final List<Integer> standardVariableIds, final String programUUID) {
+		final List<StandardVariable> standardVariables = new ArrayList<>();
+		if (standardVariableIds != null && !standardVariableIds.isEmpty()) {
+			for (final Integer id : standardVariableIds) {
+				final Variable variable = this.getOntologyVariableDataManager().getVariable(programUUID, id, false);
```

Comment:
You can just replace this code inside the loop with createStandardVariableWithAliasAsName(final int standardVariableId, final String programUUID);